core/src/main/java/com/opensymphony/xwork2/ognl/OgnlValueStack.java

```diff
@@ -482,12 +482,22 @@ public void clearContextValues() {
         ((OgnlContext) context).getValues().clear();
     }
 
+    @Deprecated
     public void setAcceptProperties(Set<Pattern> acceptedProperties) {
-        securityMemberAccess.setAcceptProperties(acceptedProperties);
+        throw new UnsupportedOperationException("Please use #useAcceptProperties instead!");
```

Comment:
Doesn't this defeat the purpose of deprecating and retaining these methods? May as well delete them if they're not functional

Reply:
My idea was to give a clear signal that these methods shouldn't be used anymore as they can degrade security. I will remove them in the [next version](https://issues.apache.org/jira/browse/WW-5328)

Reply:
I'd personally just delete them if that's the signal you want to give. Deprecating methods traditionally means they can still be used and will redirect to the replacing method where possible.

Reply:
I will keep them operational till the next version